src/v/cluster/health_monitor_types.h

```diff
@@ -194,8 +203,13 @@ using force_refresh = ss::bool_class<struct hm_force_refresh_tag>;
  */
 
 struct get_node_health_request {
```

Comment:
I think this should also be added to `get_cluster_health_request` (really, all requests that return `partition_status` but I think these are the only two)

Reply:
Yeah, don't know how I missed it)
Fixed in new version